recipes/moltenvk/all/conandata.yml

```diff
@@ -24,18 +27,32 @@ sources:
     url: "https://github.com/KhronosGroup/MoltenVK/archive/v1.1.0.tar.gz"
     sha256: "0538fa1c23ddae495c7f82ccd0db90790a90b7017a258ca7575fbae8021f3058"
 patches:
+  "1.1.11":
+    - patch_file: "patches/0003-vulkan-alias-private-extern-1.1.11.patch"
+      patch_description: "Fix vulkan alias symbols when vulkan symbols are hidden"
+      patch_type: "backport"
+      patch_source: "https://github.com/KhronosGroup/MoltenVK/pull/1718"
```

Comment:
Any chance we can go with their patch https://github.com/KhronosGroup/MoltenVK/pull/1719

this one was not accepted 🙈

Reply:
I don't know yet. This patch may not work for previous versions of MoltenVK.

Reply:
I do not think we should backport the feature especially with a different version upstream. I would be willing to approve the upstream patch, support some older version is much better than the original 👍